packages/pyright-scip/src/mainImpl.ts

```diff
@@ -174,5 +174,3 @@ export function main(argv: string[]): void {
     });
     command.parse(argv);
 }
-
-main(process.argv);
```

Comment:
This threw error for before I was able to call `main` in tests, so I deleted in temporarily.

Reply:
This `main` call was incorrect, it should've been removed earlier, as there is already a call to `main` in `main.ts` which I thought would be the entry-point for the indexer itself, whereas `test-main.ts` would be the entry-point for the test code.

Reply:
I'm confused a little bit here... This patch removes both calls to `main`. That way, running `node ./index.js --help` doesn't print the help for the indexer itself...

Normally, in C++, I'd end up with two different binaries (one for releases, and one for tests) with two different `main` functions. The test binary would call the `mainImpl` from a shared library which contains 99% of the code.

Can we do something similar here/would that be unidiomatic?